### ISSUE TITLE
walk into ocean 20 min starting.

### ISSUE THREAD

**Issue by Varamochi** [finndev/PokeBuddy#560]
I set the start place at Union Saure in San Fransisco.
It used to work very well for over 24 hours without any problem. over 70K ex/hr without evolving
Now it seems it cannot circulate back, and just keep going and going till it get to a place without any pokestop available.

**Comment by Varamochi** [comment]
Now I have to reset every 20 or 30 min to relocate it to the original place!. 
Please help.

**Comment by joejoejoey04** [comment]
Pretty sure it's to do with the fact that pokestops take forever to load in/not at all. Also, you are super lucky for it to run 24 hours. Mine has crashed with 'UI Updating' after about an hour, every single time.

**Comment by WilliamcaFox** [comment]
This seems to be a bug with the recent update. Possibly not recognizing the nearest pokestop / pokestops have refreshed? There was an update to the android app recently. This is not entirely impossible.

**Comment by Varamochi** [comment]
yes I think that is a bug since the recent update. It does not navigate back in circle.
